test/minifiedEquippable.ts

```diff
@@ -0,0 +1,250 @@
+// Note: This is just a copy of the equippable test suit with the additional tests
+// for ERC721 and nestable behavior, but using MinifiedEquippable instead.
+
+import { Contract } from 'ethers';
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import {
+  addAssetEntryEquippablesFromMock,
+  addAssetToToken,
+  mintFromMock,
+  nestMintFromMock,
+  nestTransfer,
+  parentChildFixtureWithArgs,
+  transfer,
+} from './utils';
+import { setupContextForParts } from './setup/equippableParts';
+import { setupContextForSlots } from './setup/equippableSlots';
+import shouldBehaveLikeEquippableAssets from './behavior/equippableAssets';
+import shouldBehaveLikeEquippableWithParts from './behavior/equippableParts';
+import shouldBehaveLikeEquippableWithSlots from './behavior/equippableSlots';
+import shouldBehaveLikeNestable from './behavior/nestable';
+import shouldBehaveLikeMultiAsset from './behavior/multiasset';
+import shouldBehaveLikeERC721 from './behavior/erc721';
+import {
+  RMRKCatalogMock,
+  RMRKMinifiedEquippableMock,
+  RMRKEquipRenderUtils,
+  RMRKMultiAssetRenderUtils,
+} from '../typechain-types';
+// --------------- FIXTURES -----------------------
+
+async function partsFixture() {
+  const catalogSymbol = 'NCB';
```

Comment:
Not really relevant, but I suppose the `B` comes from `Base`, right? We could change this to:
```suggestion
  const catalogSymbol = 'NCC';
```